lib/smart_answer_flows/uk-benefits-abroad.rb

```diff
@@ -743,7 +743,6 @@ def define
 
       outcome :jsa_eea_going_abroad_maybe_outcome
       outcome :jsa_ireland_outcome
-      outcome :jsa_channel_islands_outcome
```

Comment:
I believe we should fix this up to the commit where this code became obsolete; again, it makes it easier to see the impact of code changes :)